### PR TITLE
Feature-Joseph:Welcome-Email-API

### PR DESCRIPTION
**Pull request by Olayanju-1234**
feature-Joseph-welcome-email

### CODE REVIEW

**Review by Remi-dee — APPROVED** [review]
Seen

File: Developers/app/api/welcomeEmail/route.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { NextRequest, NextResponse } from "next/server";
+import nodemailer from "nodemailer";
+
+export async function POST() {
+  try {
+    const email = await NextRequest.body;
+
+    const transporter = nodemailer.createTransport({
```
Comment (annonymousauthority) [inline Developers/app/api/welcomeEmail/route.js:8]: We'll be using 
`NEXT_PUBLIC_[ENVIRONMENT VARIABLE NAME]`

Let's also stay consistent with the EV naming, so communicate with other developers working with same Variables.

File: Developers/app/api/welcomeEmail/route.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      host: process.env.EMAIL_HOST,
+      port: process.env.EMAIL_PORT,
+      auth: {
+        user: process.env.EMAIL_USERNAME,
+        pass: process.env.EMAIL_PASSWORD,
+      },
+    });
+
+    const mailData = {
```
Comment (annonymousauthority) [inline Developers/app/api/welcomeEmail/route.js:17]: We'll be using a handlebar template.
This configuration doesn't work for that. Can you change this?

If you're having difficulty, you can reach out.

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Pretty basic.
Doesn't feel like much thought was put into it.

let's fix this!

File: Developers/app/api/welcomeEmail/route.js (hunk 1 above)
Comment (Olayanju-1234) [reply Developers/app/api/welcomeEmail/route.js:16]: I have made corrections to the environment variables naming. They all follow the required pattern now.

File: Developers/app/api/welcomeEmail/route.js (hunk 2 above)
Comment (Olayanju-1234) [reply Developers/app/api/welcomeEmail/route.js:27]: I made my handlebar file and linked it to the code in my local environment. I created an EV for the path of the template that might be added later.

**Pull request closed without merging**